<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>544</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>544</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>544</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #e8edea</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="AlbumImage" name="albumImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artist">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>24</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="songName">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>71</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>32</y>
     <width>13</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>13</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>13</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/img/pause_normal.png);}
QToolButton:hover{border-image: url(:/img/pause_clicked.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="seeker">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>90</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {

background: #727982;
height: 5px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 5px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 5px;
border-radius: 4px;
}

QSlider::handle:horizontal {
width:0px
}

QSlider::handle:horizontal:hover {
width: 0px
}

QSlider::sub-page:horizontal:disabled {
background: #94ad6b;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
width: 0px
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>130</y>
     <width>181</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="likeButton">
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{border-image: url(:/img/like.png);}
QToolButton:hover{border-image: url(:/img/unlike.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="trashButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{border-image: url(:/img/trash_can_normal.png);}
QToolButton:hover{border-image: url(:/img/trash_can_clicked.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="nextButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{border-image: url(:/img/next_normal.png);}
QToolButton:hover{border-image: url(:/img/next_clicked.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="VolumeTimePanel" name="volumeTime" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="userLogin">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>70</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>未登录</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="album">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>47</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="LyricWidget" name="lyricWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>324</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="ChannelWidgetTrigger" name="channelWidgetTrigger" native="true">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>0</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="LyricWidgetTriggerLeft" name="lyricWidgetTriggerLeft" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>21</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="LyricWidgetTriggerRight" name="lyricWidgetTriggerRight" native="true">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>0</y>
     <width>31</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeTimePanel</class>
   <extends>QWidget</extends>
   <header>volumetimepanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LyricWidget</class>
   <extends>QWidget</extends>
   <header>lyricwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AlbumImage</class>
   <extends>QLabel</extends>
   <header>albumimage.h</header>
  </customwidget>
  <customwidget>
   <class>ChannelWidgetTrigger</class>
   <extends>QWidget</extends>
   <header>channelwidgettrigger.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LyricWidgetTriggerLeft</class>
   <extends>QWidget</extends>
   <header>lyricwidgettrigger.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LyricWidgetTriggerRight</class>
   <extends>QWidget</extends>
   <header>lyricwidgettrigger.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
